BLSQ/iaso
# HOTFIX: POLIO-1171: base lqas status on scope

Explain what problem this PR is resolving

Related JIRA tickets : POLIO-1171

## Self proofreading checklist


## Changes

- Calculate country status for round using scope as denominator for success ratio, i.o. total of districts with lqas data:
    - LqasSummary: use scope.length as denominator to compute country status
- Refactor `LqasSummary` to show the number of districts in scope 

## How to test

Explain how to test your PR.

From main branch, go to LQAS country view, choose a campaign
Checkout to this branch: go to the same campaign: if there were districts with no data, the success ratio should be lower.
Note: to test locally, you may have to run `copy_data_store.py` so you have lqas endpoints


## Print screen / video
Before
![Screenshot 2023-09-01 at 17 39 03](https://github.com/BLSQ/iaso/assets/38907762/52568768-a5d0-48bd-a02f-70399c118c04)
After
![Screenshot 2023-09-01 at 17 38 39](https://github.com/BLSQ/iaso/assets/38907762/e8a8c90b-175d-4f54-ab33-7b890499847d)




## Notes

This is a temporary fix until a long term solution is implemented. See ticket POLIO-1171
This PR can be deployed as hotfix
